release-0_11: Make install gives error

```
~/grpc-test$ sudo make install
[INSTALL] Installing public C headers
[MAKE]    Generating cache.mk
[STRIP]   Stripping libgpr.a
strip: '/usr/local/google/home/stanleycheung/grpc-test/objs/opt/src/core/support/libs/opt/libgpr.a'\
: No such file
make: *** [strip-static_c] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -49,7 +49,7 @@ SYSTEM = MINGW32
 endif
 
 
-MAKEFILE_PATH = $(abspath $(lastword $(MAKEFILE_LIST)))
+MAKEFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
 ifndef BUILDDIR
 BUILDDIR_ABSOLUTE = $(patsubst %/,%,$(dir $(MAKEFILE_PATH)))
 else

--- a/templates/Makefile.template
+++ b/templates/Makefile.template
@@ -65,7 +65,7 @@
   endif
 
 
-  MAKEFILE_PATH = $(abspath $(lastword $(MAKEFILE_LIST)))
+  MAKEFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
   ifndef BUILDDIR
   BUILDDIR_ABSOLUTE = $(patsubst %/,%,$(dir $(MAKEFILE_PATH)))
   else

